packages/react-storage/src/components/StorageBrowser/context/elements/DropZoneElement.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { ViewElement, ViewElementProps } from './definitions';
+import { useDropZone } from '@aws-amplify/ui-react-core';
+import { CLASS_BASE } from '../../views/constants';
+
+export interface DropZoneElementProps extends ViewElementProps {
+  handleDroppedFiles: (files: File[]) => void;
+}
+
+export const DropZoneElement = ({
+  children,
+  handleDroppedFiles,
+}: DropZoneElementProps): React.ReactElement => {
+  const [isDragging, setIsDragging] = React.useState(false);
+
+  const { dragState, ...dropHandlers } = useDropZone({
```

Comment:
Don't see `dragState` used anywhere, but afaik we should be able to remove the entirety of this code and just use the value of `dragState` to determine whether the `Table` should show the drag active styling

Reply:
Updated to remove `isDragging` state and `dropHandlers` and just use `dragState` to set styling when files are dragged over the table

Reply:
Restored `dropHandlers` in a later commit. I had thought it wasn't necessary due to a cached state in the example app I was testing in